xbmc/addons/AddonManager.cpp

```diff
@@ -607,6 +607,8 @@ bool CAddonMgr::FindAddon(const std::string& addonId,
   FindAddons(installedAddons, "special://xbmcbin/addons");
   FindAddons(installedAddons, "special://xbmc/addons");
   FindAddons(installedAddons, "special://home/addons");
+  FindAddons(installedAddons, "special://xbmcbinaddons");
```

Comment:
...therefor i do not see the need to include `special://xbmcbinaddons`
because this is also mapped to: `/opt/matrix/lib/kodi/addons`